src/test/kotlin/org/rust/lang/core/resolve/RsPackageLibraryResolveTest.kt

```diff
@@ -556,8 +556,9 @@ class RsPackageLibraryResolveTest : RsResolveTestBase() {
         }
     """)
 
+    // FIXME
     @MockEdition(CargoWorkspace.Edition.EDITION_2018)
-    fun `test resolve bang proc definition in macro crate itself`() = stubOnlyResolve("""
+    fun `ignored test resolve bang proc definition in macro crate itself`() = stubOnlyResolve("""
```

Comment:
I'd prefer to use `expect<SomeExpcetion>` instead of `ignore`. One day a test can be fixed unintentionally. In this case, an `expect` test will fail and the developer will be forced to remove `expect` from the test. `ignored` can be just missed.

Reply:
Done